wpengine/lostromos
# Add integration tests and documentation

Signed-off-by: Derek Rushing <derek.rushing21@gmail.com>

# What Are We Doing Here

Adding integration tests and documentation. Also moves the manual_testing.md file in to the development markdown, and adds a new Testing.md file for tracking our testing implementation/expectations.

## How to Verify

1. Stand up a minikube service.
2. Check out this PR and run ./test-scripts/integration-tests.sh
